contrib/powershell/Monitor-Room-Resources.ps1

```diff
@@ -0,0 +1,40 @@
+# Set the folder paths
+$sourceFolder = "../../resource/rooms" # Change this to your source folder path
```

Comment:
This is kind of a weird relative path.  Should you instead assume the top of the source tree?

Reply:
I ended up adding some additional lines to determine the proper paths, so that it can be run from anywhere.